src/kodeknekkerne/index.md

```diff
@@ -1,22 +1,8 @@
-﻿---
+---
 title: Kodeknekkerne
-external: https://kodeknekkerne.kodegenet.no/
 language: nb
+external: https://kodeknekkerne.kodegenet.no/
+tags:
+    topic: [block_based, step_based]
+    grade: [primary, secondary, junior]
 ---
-
-NRK Skole har utviklet denne serien som er en kombinasjon av gameshow og

```

Comment:
Kan vel bare bruke den funksjonen som allerede ligger inne for hover på oppgaver? :)